Source/CarShack/CarShack.csproj

```diff
@@ -32,7 +32,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\WebApi.HypermediaExtensions\WebApi.HypermediaExtensions.csproj" />
+    <ProjectReference Include="..\WebApi.HypermediaExtensions\RESTyard.WebApi.Extensions.csproj" />
```

Comment:
We should adopt the folder structure too. `..\WebApi.HypermediaExtensions`-> `..\RESTyard\AspNetCore\Extensions`?

Reply:
Folder structure was not adopted now because of potential open work which might cause massive conflicts.